website/static/js/filesWidget.js

```diff
@@ -0,0 +1,77 @@
+var $ = require('jquery');
```

Comment:
filesWidget is a decoupling of the project files widget from the project page JS, and does get reused in registrationEditorExtensions.js